modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/display/context/CommerceOrderContentDisplayContext.java

```diff
@@ -771,6 +773,19 @@ public List<HeaderActionModel> getHeaderActionModels()
 					transitionName = "submit";
 				}
 			}
+			else if ((commerceOrderStatus.getKey() ==
+						CommerceOrderConstants.ORDER_STATUS_QUOTE_REQUESTED) &&
+					 isRequestQuoteEnabled()) {
```

Comment:
ok for now, will have to extend this isRequestQuoteEnabled() method so that it will check for "quote only" products in the cart

Reply:
@luca-pellizzon , this is not within the scope of COMMERCE-11297 yes? I updated everything else as requested.